src/net_processing.h

```diff
@@ -29,6 +29,8 @@ struct CNodeStateStats {
     int m_starting_height = -1;
     std::chrono::microseconds m_ping_wait;
     std::vector<int> vHeightInFlight;
+    bool fRelayTxes{false};
+    CAmount minFeeFilter{0};
```

Comment:
eeb0e84fc5c259dfd00f7673ad4a51582b100282: Why not `-1` (an impossible number), like the height values above? Or  maybe leave it uninitialized to make memory sanitizers more useful?

Reply:
As above (https://github.com/bitcoin/bitcoin/pull/21160#discussion_r715508638)